pubtools/_quay/container_image_pusher.py

```diff
@@ -128,11 +128,16 @@ def _prepare_dest_refs(self, push_item):
         for repo, tags in sorted(push_item.metadata["tags"].items()):
             internal_repo = get_internal_container_repo_name(repo)
             for tag in tags:
+                dst_tag = (
+                    f"{tag}.{push_item.metadata.get('com.redhat.pre-release')}"
+                    if push_item.metadata.get("com.redhat.pre-release")
```

Comment:
Can we use "com.redhat.prerelease"  label instead? ET is also using the same to identify the bundle

Reply:
I thought it hasn't been settled yet, I don't see it in anywhere in ET code and also I haven't seen any pre-release build with that label, so this this can change I assume

Reply:
I haven't checked ET code thoroughly, but in the testing instructions added in CWFCONF jira the label mentioned was "com.redhat.prerelease" I have linked the comment on CLOUDDST-19629